x-pack/test/functional_enterprise_search/apps/enterprise_search/with_host_configured/app_search/engines.ts

```diff
@@ -33,12 +35,28 @@ export default function enterpriseSearchSetupEnginesTests({
     });
 
     describe('when an enterpriseSearch host is configured', () => {
-      it('navigating to the enterprise_search plugin will redirect a user to the App Search root', async () => {
+      it('navigating to the enterprise_search plugin will redirect a user to the App Search Engines Overview page', async () => {
+        await security.user.create('enterprise_search', {
+          password: 'changeme',
+          roles: ['kibana_admin'],
+          full_name: 'enterprise_search',
+        });
+
+        await PageObjects.security.forceLogout();
+        await PageObjects.security.login('enterprise_search', 'changeme', {
+          expectSpaceSelector: false,
+        });
+
         await PageObjects.enterpriseSearch.navigateToPage();
         await retry.try(async function() {
           const currentUrl = await browser.getCurrentUrl();
           expect(currentUrl).to.contain('/app_search');
         });
+        browser.getActions().pause(5000);
```

Comment:
Is there a more concrete condition we can wait for here?

Reply:
Sorry, I checked this in in a premature state to show my progress.

I'm not relying on that, I had put that in temporarily in an attempt to inspect the page in chrome manually.

Reply:
👍 And, in hindsight, my minor comment was likely premature for a Draft PR. I'll try to keep Draft PR feedback more focused on the big picture.

Reply:
No no, I'm actually glad you dropped that comment. I think it's acceptable to look at a Draft PR and notice concerning code. Like it's probably better to catch things like that early before it gets out of hand. Like if I started added pauses like that all over the code it would be a pretty terrible decision.